ds/ds.php

```diff
@@ -1875,6 +1875,16 @@ public function xor(Map $map): Map
      */
     class Pair implements JsonSerializable
     {
+        /**
+         * @var mixed
+         */
+        public $key = null;
```

Comment:
The `= null` is useless

Reply:
It follows [implemented behavior](https://github.com/php-ds/ext-ds/blob/master/src/php/classes/php_pair_ce.c#L66-L67)?

Reply:
null is the default value anyway so it means exactly the same, adding = null to a property only makes sense when using typed properties which is not the case here